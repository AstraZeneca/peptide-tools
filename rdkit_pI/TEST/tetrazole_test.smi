CCCc1nnn[nH]1