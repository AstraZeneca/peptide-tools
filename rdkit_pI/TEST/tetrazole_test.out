[16:05:45] WARNING: no name column found on line 0
 
======================================================================================================================================================
pI
---------------------------------
     pI mean  6.0         
         err  0.0         
         std  0.0         
 IPC_peptide  6.0         
     ProMoST  6.0         
 Gauci_calib  6.0         
  Bjellqvist  6.0         
     Rodwell  6.0         
    Grimsley  6.0         
   Thurlkill  6.0         
     Solomon  6.0         
   Lehninger  6.0         
      EMBOSS  6.0         
 
 
======================================================================================================================================================
Q at pH7.4
---------------------------------
Q at pH7.4 mean  -0.63       
         err  0.0         
         std  0.0         
 IPC_peptide  -0.63       
     ProMoST  -0.63       
 Gauci_calib  -0.63       
  Bjellqvist  -0.63       
     Rodwell  -0.63       
    Grimsley  -0.63       
   Thurlkill  -0.63       
     Solomon  -0.63       
   Lehninger  -0.63       
      EMBOSS  -0.63       
 
 
pH interval with charge between -0.2 and  0.2 for pKa set: IPC_peptide and prediction tool: modified Dimorphite-DL
 5.5 -  6.6
 
List of calculated BASE pKa's with the corresponding fragments
smiles or AA, base pka : CCCc1nnn[nH]1    4.4
smiles or AA, base pka : CCCc1nnn[nH]1    4.4
smiles or AA, base pka : CCCc1nnn[nH]1    4.4
 
List of calculated ACID pKa's with the corresponding fragments
smiles or AA, acid pka : CCCc1nnn[nH]1    7.2
 
List of constantly ionized fragments with the corresponding fragments
